### ISSUE TITLE
Fine-tune yolov8l-face.pt model

### ISSUE THREAD

**Issue by DarsithSPillai** [ultralytics/ultralytics#18279]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I want to fine-tune the yolov8l-face.pt model using 5,000 images that I have. Should I use this command?
`yolo task=detect mode=train model="yolov8l-face.pt" data=data.yaml`

After fine-tuning with 5,000 images, if I want to further improve my trained model using an additional 3,000 images, what should I do?

Can I simply add the new 3,000 images to the dataset and use the following command?
`yolo task=detect mode=train model="path/to/previous/train/best.pt" data=data.yaml`

Please help me. it's argent.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @DarsithSPillai, thank you for your interest in Ultralytics 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com) where you'll find comprehensive tutorials and examples on [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage, which may already answer your question.

If this is a custom training ❓ Question, it's great to hear you're refining your YOLO model with additional images! Please share more details to help us assist you better:
- The structure of your dataset (e.g., is `data.yaml` configured correctly?).
- Logs or outputs created after running your training command.
- Versions of `ultralytics`, Python, and PyTorch you're using.

Additionally, you may want to verify that you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to ensure optimal performance while training with both your initial and new datasets.

Join the Ultralytics community where it suits you best. For quick responses and discussions, jump into [Discord](https://discord.com/invite/ultralytics) 🎧. For extensive debate or solutions, visit [Discourse](https://community.ultralytics.com). Or explore the latest posts and ideas on our [Subreddit](https://reddit.com/r/Ultralytics).

## Upgrade

Please ensure you're using the latest version of the `ultralytics` package. Upgrade it (along with all dependencies) in a supported environment ([Python>=3.8](https://www.python.org/) and [PyTorch>=1.8](https://pytorch.org/get-started/locally/)) to ensure compatibility and resolve potential issues:

```bash
pip install -U ultralytics
```

## Environments

YOLO runs smoothly in multiple environments. Try out any of these verified options:
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** - Refer to the [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/).
- **Amazon** - See the [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/).
- **Docker Image** - Learn from the [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>.

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing. CI tests run daily and verify that all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) work correctly across macOS, Windows, and Ubuntu.

🔔 This is an automated response to guide you while an Ultralytics engineer reviews your issue and provides a detailed response soon.

**Comment by Y-T-G** [comment]
You can follow this.

https://www.reddit.com/r/Ultralytics/comments/1eolwl8/the_correct_way_to_train_from_a_previously/

**Comment by DarsithSPillai** [comment]
I want to fine-tune the face detection model [yolov8l-face.pt](https://github.com/akanametov/yolo-face/releases/download/v0.0.0/yolov8l-face.pt) using 5,000 images that I have.
How can I do that?

**Comment by Y-T-G** [comment]
You need to get the original dataset used to train it

**Comment by DarsithSPillai** [comment]
Thanks for the response, @Y-T-G 

Do I need to combine both the original dataset used to train the yolov8l-face.pt model and my 5,000 images into a new dataset for fine-tuning?

**Comment by Y-T-G** [comment]
Yes

**Comment by DarsithSPillai** [comment]
@Y-T-G 
Thanks!

The yolov8l-face.pt model is trained on the WIDER FACE dataset, so I need to combine this dataset with my 5,000 images for fine-tuning.

After fine-tuning the yolov8l-face.pt model with these datasets, I will get a new model (best.pt). If I want to fine-tune this best.pt model with an additional 3,000 images, do I need to use the previous WIDER FACE dataset, the 5,000 images, and the new 3,000 images as the combined dataset?

**Comment by Y-T-G** [comment]
You will always have to combine

**Comment by DarsithSPillai** [comment]
@Y-T-G  

Okay,  
Is this command used for fine-tuning the model?  
`yolo task=detect mode=train model="yolov8l-face.pt" data=data.yaml`

**Comment by Y-T-G** [comment]
> You can follow this.
> 
> https://www.reddit.com/r/Ultralytics/comments/1eolwl8/the_correct_way_to_train_from_a_previously/

Set the hyperparameters according to here
